setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='vdist',
-    version='0.5',
+    version='1.0',
```

Comment:
I feel this version quite polished and ready to be broadly used, 1.0 sounded great but we can put whatever version you like. :-)